massa-bootstrap/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+async-speed-limit = { git = "https://github.com/adrien-zinger/async-speed-limit", rev = "36d79e0", features = ["default", "tokio"] }
```

Comment:
From our last discussion I understood that the original repo contains all the necessary features. I'm surprised to see a fork being used here.

Reply:
It's because we're never sure that the repository will be removed or not. I can change that easily but I thought it was better if I control that before they publish the feature

Reply:
Should I make a x1000 ?

Reply:
what's the feature that hasn't been published ?

Reply:
okay